ports/nrf/peripherals/nrf/nvm.c

```diff
@@ -40,6 +40,8 @@
 
 STATIC bool sd_is_enabled(void) {
     uint8_t sd_en = 0;
+    if (__get_PRIMASK())
```

Comment:
this now look a bit odd with latest push, I mean we shouldn't call this funciton within critical section at all. In fact, I think we don't ever disable CPSR-I when SD is enabled, and should use their sd_* API instead.

Reply:
This was added due to tinyusb calling `sd_flash_operation_wait_until_done()` which calls `sd_is_enabled()`, which didn't work when interrupts were disabled.

Reply:
ah, sorry, this is another issue out of my radar.